src/ctl/plugins/version_base.py

```diff
@@ -117,6 +123,12 @@ def add_arguments(cls, parser, plugin_config, confu_cli_args):
             choices=["major", "minor", "patch", "prerelease"],
             help="bumps the specified version segment by 1",
         )
+        op_bump_parser.add_argument(
```

Comment:
@solababs how does the default work here? Is the default to do a tag or not?

I think we want to keep the default to do a tag, and have an argument to disable it. 

Then this also needs to be added to the usage docs.